backend/api/routes/index.js

```diff
@@ -8,5 +8,7 @@ router.post('/preferences/repos', api.updateRepos);
 router.post('/preferences/metric-types', api.updateMetricTypes);
 router.post('/unsubscribe', api.unsubscribe);
 router.post('/resubscribe', api.resubscribe);
+router.post('/delete-account', api.destroy);
+router.post('/cancel-deletion', api.cancelDestruction);
```

Comment:
Use REST

Reply:
This refactoring will take place at a later stage and will include all routes (not just those related to this PR).

Reply:
Addressed in [f290b16](https://github.com/meeshkan/vanity/commit/f290b16908b97f6819a40db530a9bbef889ee928).